### PR TITLE
Add repo readme indexing all services

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Overview
+This repository comprises prebuilt healthcare-related services that are ready for immediate use. Users have the flexibility to deploy and run these services in their respective environments. Optionally, these services can be deployed in [Choreo](https://console.choreo.dev/home) as well. For further details, refer to the documentation of the individual services.
```
Comment (sameeragunarathne) [inline README.md:2]:
```suggestion
This repository comprises pre-built healthcare-related services, designed for instant use without without the need for any modifications. Users have the flexibility to deploy and run these services in their respective environments. Optionally, these services can be deployed in [Choreo](https://console.choreo.dev/home) as well. For further details, refer to the documentation of the individual services.
```

Comment (sameeragunarathne) [reply README.md:2]: WDYT on the above suggestions?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+#### [SMART Configuration Service](conformance/smart-config-fhirr4-service/)
+This service can be used to expose the discovery document containing authorization endpoint URLs and SMART-features supported by a FHIR server.
+
+### Transformation
+#### [HL7v2 to FHIR](transformation/v2-to-fhirr4-service/)
+Transform HL7v2 messages into FHIR resources using this service.
+
+#### [CCDA to FHIR](transformation/ccda-to-fhirr4-service/)
+This service can be usde to transform C-CDA messages into FHIR resources.
```
Comment (sameeragunarathne) [inline README.md:17]:
```suggestion
This service can be used to transform C-CDA messages into FHIR resources.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- [Epic FHIR R4 Administration Service](ehr-connectivity/epic-fhirr4-administration-api-service/)
+- [Epic FHIR R4 Clinical Service](ehr-connectivity/epic-fhirr4-clinical-api-service/)
+- [Epic FHIR R4 Diagnostics Service](ehr-connectivity/epic-fhirr4-diagnostics-api-service/)
+- [Epic FHIR R4 Financial Service](ehr-connectivity/epic-fhirr4-financial-api-service/)
+- [Epic FHIR R4 Medications Service](ehr-connectivity/epic-fhirr4-medications-api-service/)
+- [Epic FHIR R4 Workflow Service](ehr-connectivity/epic-fhirr4-workflow-api-service/)
+
+### Miscellaneous
+#### [Authorization Service](miscellaneous/authz-fhirr4-service/)
+Apply basic patient and privileged user-based authorization policies for FHIR APIs using this service.
```
Comment (sameeragunarathne) [inline README.md:32]: This service basically evaluate the policies?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+#### [Audit Service](miscellaneous/audit-service/)
+Record audit events upon calling FHIR APIs using this service.
+
+#### [FHIR Path Service](miscellaneous/fhirpath-service/)
+Evaluate FHIR path expressions against a FHIR payload using this service.
```
Comment (sameeragunarathne) [inline README.md:38]: Shall we rephrase the sentences with "Use this service" eg: "Use this service to evaluate FHIR path expressions against a FHIR payload." WDYT?

File: README.md (hunk 1 above)
Comment (ThishaniLucas) [reply README.md:2]: I feel the above already implies this. Doesn't it?

File: README.md (hunk 3 above)
Comment (ThishaniLucas) [reply README.md:32]: Yes. And we can use this evaluation to secure a FHIR API. So ultimately the policies get applied to the API.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+## Overview
+This repository comprises prebuilt healthcare-related services that are ready for immediate use. Users have the flexibility to deploy and run these services in their respective environments. Optionally, these services can be deployed in [Choreo](https://console.choreo.dev/home) as well. For further details, refer to the documentation of the individual services.
+
```
Comment (nirmal070125) [inline README.md:3]: Shall we add a pre-requisite section? mention the software requirements etc.

File: README.md (hunk 4 above)
Comment (ThishaniLucas) [reply README.md:38]: Any reason for that? Both imply same meaning ryt?

File: README.md (hunk 5 above)
Comment (ThishaniLucas) [reply README.md:3]: added

File: README.md (hunk 1 above)
Comment (sameeragunarathne) [reply README.md:2]: +1. prebuilt should be pre-built

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**